public/app/core/components/AppChrome/AppChrome.tsx

```diff
@@ -152,6 +164,17 @@ const getStyles = (theme: GrafanaTheme2, searchBarHidden: boolean) => {
           display: 'block',
         },
       },
+      config.featureToggles.bodyScrolling
+        ? {
+            position: 'fixed',
+            height: `calc(100% - ${searchBarHidden ? TOP_BAR_LEVEL_HEIGHT : TOP_BAR_LEVEL_HEIGHT * 2}px)`,
+            zIndex: 2,
+          }
+        : {
+            zIndex: theme.zIndex.navbarFixed,
+          }
+    ),
+    sidePaneLeft: css(
```

Comment:
maybe we should set `width: MENU_WIDTH` here?

Reply:
yea, ideally the white styling and border should be set here as well